### PR TITLE
Fix golint failure in pkg/kubectl/cmd/certificates

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
This fixes golint failures of the following file:

- pkg/kubectl/cmd/certificates/

**Which issue(s) this PR fixes**:
Ref [#68026] (https://github.com/kubernetes/kubernetes/issues/68026)
Ref [#79782] (https://github.com/kubernetes/kubernetes/pull/79782)
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```
None
```

### CODE REVIEW

**Comment by ZP-AlwaysWin** [comment]
/assign @deads2k

hi @deads2k

Best wishes

**Comment by ZP-AlwaysWin** [comment]
/assign @draveness

Hi @draveness

Because the  #79782 commit was submitted many times and there were too many conflicts, I closed it and resubmitted it.

Thank for review my first pull request for k8s。
Thank you very much
Best wishes

File: pkg/kubectl/cmd/certificates/certificates.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
 )
 
-func NewCmdCertificate(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+func newCmdCertificate(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
```
Comment (neolit123) [inline pkg/kubectl/cmd/certificates/certificates.go]: this function is called from another package so cannot be lowercased:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/cmd.go#L494

you need to add a export comment on top of it instead.
example:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/cmd.go#L411

**Pull request closed without merging**